Add MarkerUpdates to database

navilperez opened the pull request:
Functions for adding and retrieving MarkerUpdates (#39) from the database.

navilperez committed:
Add MarkerUpdates to database.

Functions for adding and retrieving MarkerUpdates from the database.

rachel-fenichel commented on blockly-rtc/server/Database.js:
```diff
@@ -174,6 +174,80 @@ class Database {
   };
+
+  /**
+   * Add client to clients if it is not already in the table.
```
This isn't called in this PR.

navilperez replied:
This function is not yet called. It will be called by MarkerManager when a client first connects to the server.
The purpose is to ensure that a client is in clients table otherwise the SQL queries updating the client's markerLocation will throw errors if it has not been added to the table.

navilperez committed:
Middleware for MarkerUpdates.

APIs and server functions for sending MarkerUpdates between the client
and server.

navilperez committed:
Revert "Middleware for MarkerUpdates."

This reverts commit 4dab3640ffd55631564a0728ea9f442b01d66ec8.
